browser/src/layer/tile/TileLayer.TableOverlay.js

```diff
@@ -324,6 +333,57 @@ L.CanvasTileLayer.include({
 		e.target.dragging.freezeY(true);
 	},
 
+	_addInsertMarkers: function(type, start) {
+		const markersDistanceFromTable = 10;
+		const delta1 = this._convertPixelToTwips(markersDistanceFromTable);
+		
+		let documentPosition;
+		let markerWidth;
+		let markerHeight;
+		
+		if (type === 'column') {
+			const tableRightEdge = parseInt(this._currentTableData.columns.right);
+			const columnPosition = this._tablePositionColumnOffset + tableRightEdge + delta1;
+			documentPosition = { pX: columnPosition, pY: start };
```

Comment:
Document position in other examples is SimplePoint from start. pX and pY are used for "core pixel X" and "core pixel Y". Here there are used as twips positions. We need to either change the naming to "x" and "y" only, or use SimplePoint as document position and change constructor of the new section to accept SimplePoint.